bodhi/server/models.py

```diff
@@ -1653,7 +1653,11 @@ def beautify_title(self, amp=False):
         &amp; html entity
         """
         if len(self.builds) > 2:
```

Comment:
You could simplify this a bit by introducing an inline function for how to pick the label to do a little less if/elsing. Something like this:

```
def build_label():
    return build.nvr if nvr else packagename_from_nvr(self, build.nvr)
```

Then you can just change the calls to ```packagename_from_nvr``` to ```build_label```.

It's fine to keep it the way it is if you prefer, just a suggestion.